lib/core/renderer.js

```diff
@@ -472,3 +472,18 @@ export const resourceMap = [
     transform: parseTemplate
   }
 ]
+
+const retrieveCompressionMiddleware = ({ compressor, gzip }) => {
```

Comment:
This part is here better or in options.js ?

Reply:
Agreed!